examples/springdoc-openapi-scala-2/simple/src/main/scala/za/co/absa/springdocopenapiscala/examples/simple/OpenAPIConfiguration.scala

```diff
@@ -0,0 +1,45 @@
+/*
+ * Copyright 2023 ABSA Group Limited
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package za.co.absa.springdocopenapiscala.examples.simple
+
+import io.swagger.v3.oas.models.OpenAPI
+import io.swagger.v3.oas.models.info.Info
+import org.springdoc.core.customizers.OpenApiCustomizer
+import org.springframework.context.annotation.{Bean, Configuration}
+
+import za.co.absa.springdocopenapiscala.{Bundle, OpenAPIModelRegistration}
+
+@Configuration
+class OpenAPIConfiguration {
+
+  private val springDocOpenAPIScalaBundle = new Bundle(
+    Seq((openAPI: OpenAPI) =>
+      openAPI.setInfo(
+        new Info()
+          .title("Example API")
```

Comment:
```suggestion
          .title("Example API in OpenAPI v2")
```

Reply:
Makes sense, done in https://github.com/AbsaOSS/springdoc-openapi-scala/pull/27/commits/a9b43fef04020f9203d0119bf5dde067a769e230